addons/payment/static/src/xml/payement_manage_form_confirmation_dialog.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates xml:space="preserve">
+    <t t-name="payement.ManageFormConfirmationDialog" owl="1">
+        <div>
+            <span>Are you sure you want to delete this payment method?</span>
+            <t t-if="linkedRecordsInfo.length > 0">
+                <br/>
+                <span>It is currently linked to the following documents:</span>
+                <ul>
+                    <t t-foreach="linkedRecordsInfo" t-as="documentInfo" t-key="documentInfo_index">
+                        <li t-att-title="documentInfo.description" t-att-href="documentInfo.url" t-esc="documentInfo.name"/>
+                    </t>
+                </ul>
+            </t>
+        </div>
+    </t>
+</templates>
```

Comment:
There is a bunch of files you are adding without a blank newline.
Might be worth using configuring your editor to automatically fix those.